WebDriverAgentLib/Categories/XCUIElement+FBTyping.m

```diff
@@ -141,24 +142,33 @@ - (BOOL)fb_clearTextWithSnapshot:(FBXCElementSnapshotWrapper *)snapshot
   NSString *placeholderValue = snapshot.placeholderValue;
   NSUInteger preClearTextLength = [currentValue fb_visualLength];
   do {
-    if (retry >= MAX_CLEAR_RETRIES - 1) {
-      // Last chance retry. Tripple-tap the field to select its content
-
-      if ([self respondsToSelector:@selector(tapWithNumberOfTaps:numberOfTouches:)]) {
-        // e.g. tvOS 17 raised unrecognized selector error for XCUIElementTypeSearchField
-        // while following typeText worked.
-        [self tapWithNumberOfTaps:3 numberOfTouches:1];
-      }
-      return [FBKeyboard typeText:backspaceDeleteSequence error:error];
-    }
-
-    NSString *textToType = [backspaceDeleteSequence fb_repeatTimes:preClearTextLength];
     if (shouldPrepareForInput && 0 == retry) {
       [self fb_prepareForTextInputWithSnapshot:snapshot];
     }
-    if (![FBKeyboard typeText:textToType error:error]) {
+    NSString *backspacesToType = [backspaceDeleteSequence fb_repeatTimes:preClearTextLength];
+#if TARGET_OS_IOS
+    if (retry == MAX_CLEAR_RETRIES - 2) {
```

Comment:
so,

1st attempt -> typeText 
2nd -> fb_performIOHIDEventWithPage
3rd -> `tapWithNumberOfTaps:3`

correct? Or I wondered if we could try the fb_performIOHIDEventWithPage firs

Reply:
What;s about https://github.com/appium/WebDriverAgent/pull/811 ?